pkg/humio/client.go

```diff
@@ -246,18 +241,23 @@ func (c *Client) Stream(method string, path string, query Query, ch chan Streami
 		return err
 	}
 	defer func() {
-		if err := res.Body.Close(); err != nil {
```

Comment:
Is there a reason to remove this error check? IIRC it's saved us before from having issues when the body is unable to be closed.

Reply:
I can add it back but we need to think of a way to remove the logger from this file. `pkg/humio` shouldn't contain grafana stuff